### ISSUE TITLE
Kernel Dies while using conv2d layer / function

### ISSUE THREAD

**Issue by deepanshu650** [pytorch/pytorch#75265]
### 🐛 Describe the bug

Kernel dies and restarts automatically as soon as model having conv2d is used.
It happens for both - conv2d layer (torch.nn.Conv2d) and conv2d function (torch.nn.functional.conv2d), while i am able to use linear layer model in my system.
Here is a minimal example code

```
import numpy as np
import torch
import torch.nn.functional as F
from torchvision import datasets
import torch.nn as nn

class model_conv_test(nn.Module):
     def __init__(self):
         super().__init__()
         self.l1 = nn.Conv2d(1,5, (3,3))
     def forward(self, x):
         return self.l1(x)

x = torch.rand((1,1,16,16))
model_test = model_conv_test()
model_test(x)
```
<img width="751" alt="Screenshot 2022-04-05 at 9 12 31 PM" src="https://user-images.githubusercontent.com/65191985/161819396-6d0b318a-a9de-4125-8397-818efa04f163.png">

### Versions

Collecting environment information...
PyTorch version: 1.11.0
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: macOS 12.2.1 (x86_64)
GCC version: Could not collect
Clang version: Could not collect
CMake version: version 3.19.7
Libc version: N/A

Python version: 3.8.5 (default, Sep 4 2020, 02:22:02) [Clang 10.0.0 ] (64-bit runtime)
Python platform: macOS-10.16-x86_64-i386-64bit
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] botorch==0.6.0
[pip3] gpytorch==1.6.0
[pip3] numpy==1.19.2
[pip3] numpydoc==1.1.0
[pip3] torch==1.11.0
[pip3] torchvision==0.12.0
[conda] blas 1.0 mkl
[conda] botorch 0.6.0 pypi_0 pypi
[conda] gpytorch 1.6.0 pypi_0 pypi
[conda] mkl 2019.4 233
[conda] mkl-service 2.3.0 py38hfbe908c_0
[conda] mkl_fft 1.2.0 py38hc64f4ea_0
[conda] mkl_random 1.1.1 py38h959d312_0
[conda] numpy 1.19.2 py38h456fd55_0
[conda] numpy-base 1.19.2 py38hcfb5961_0
[conda] numpydoc 1.1.0 pyhd3eb1b0_1
[conda] torch 1.11.0 pypi_0 pypi
[conda] torchvision 0.12.0 pypi_0 pypi

**Comment by soulitzer** [comment]
Could you try on colab? I could not reproduce the behavior there

**Comment by deepanshu650** [comment]
@soulitzer yes i am able to run it in colab. I am able to run it in AWS sagemaker as well.
But not in my local machine.
